src/library.js

```diff
@@ -2416,6 +2416,9 @@ mergeInto(LibraryManager.library, {
     if (!warnOnce.shown) warnOnce.shown = {};
     if (!warnOnce.shown[text]) {
       warnOnce.shown[text] = 1;
+#if ENVIRONMENT_MAY_BE_NODE
+      if (ENVIRONMENT_IS_NODE) text = 'warning: ' + text;
+#endif
```

Comment:
How about doing this on `err()` itself? Say `err: ` as a prefix. Then it would help not just warnings.

Reply:
Maybe, but that would be way more wide reaching.. and I think there is also code that assuming that `err()` is a raw write the stderr.   I certainly think of `out` and `err` as `stdout` and `stderr` so I wouldn't expect them to be decorated like this.